src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt

```diff
@@ -98,36 +103,27 @@ data class RecipientCltvExpiryParams(val min: CltvExpiryDelta, val max: CltvExpi
  * @param fulfillSafetyBeforeTimeoutBlocks number of blocks necessary to react to a malicious peer that doesn't acknowledge and sign our HTLC preimages.
- * @param checkHtlcTimeoutAfterStartupDelaySeconds delay in seconds before we check for timed out HTLCs in our channels after a wallet restart.
+ * @param checkHtlcTimeoutAfterStartupDelay delay in seconds before we check for timed out HTLCs in our channels after a wallet restart.
  * @param htlcMinimum minimum accepted htlc value.
  * @param toRemoteDelayBlocks number of blocks our peer will have to wait before they get their main output back in case they force-close a channel.
  * @param maxToLocalDelayBlocks maximum number of blocks we will have to wait before we get our main output back in case we force-close a channel.
  * @param minDepthBlocks minimum depth of a transaction before we consider it safely confirmed.
  * @param feeBase base fee used in our channel_update: since our channels are private and we don't relay payments, this will be basically ignored.
- * @param feeProportionalMillionth proportional fee used in our channel_update: since our channels are private and we don't relay payments, this will be basically ignored.
- * @param revocationTimeoutSeconds delay after which we disconnect from our peer if they don't send us a revocation after a new commitment is signed.
- * @param authTimeoutSeconds timeout for the connection authentication phase.
- * @param initTimeoutSeconds timeout for the connection initialization phase.
- * @param pingIntervalSeconds delay between ping messages.
- * @param pingTimeoutSeconds timeout when waiting for a response to our ping.
- * @param pingDisconnect disconnect when a peer doesn't respond to our ping.
- * @param autoReconnect automatically reconnect to our peers.
- * @param initialRandomReconnectDelaySeconds delay before which we reconnect to our peers (will be randomized based on this value).
- * @param maxReconnectIntervalSeconds maximum delay between reconnection attempts.
- * @param chain bitcoin chain we're interested in.
- * @param channelFlags channel flags used to temporarily enable or disable channels.
+ * @param feeProportionalMillionths proportional fee used in our channel_update: since our channels are private and we don't relay payments, this will be basically ignored.
+ * @param pingInterval delay between ping messages.
+ * @param initialRandomReconnectDelay delay before which we reconnect to our peers (will be randomized based on this value).
+ * @param maxReconnectInterval maximum delay between reconnection attempts.
  * @param paymentRequestExpirySeconds our Bolt 11 invoices will only be valid for this duration.
- * @param multiPartPaymentExpirySeconds number of seconds we will wait to receive all parts of a multi-part payment.
+ * @param multiPartPaymentExpiry number of seconds we will wait to receive all parts of a multi-part payment.
```

Comment:
nit: comment still stays that it is a number of seconds

Reply:
Done (and renamed the parameter) in 7447a5cb.